docs/installing.rst

```diff
@@ -55,8 +74,69 @@ Install behind a proxy::
   python get-pip.py --proxy="[user:passwd@]proxy.server:port"
 
 
-Upgrade pip
------------
+Installing with Linux Package Managers
+--------------------------------------
+
+Fedora
+~~~~~~
+
+To install the Fedora supported version of pip:
+
+* < Fedora 23:
+
+ * Python 2: ``sudo yum install python-pip``
+ * Python 3: ``sudo yum install python3-pip``
+
+* >= Fedora 23:
+
+ * Python 2: ``sudo dnf install python-pip``
+ * Python 3: ``sudo dnf install python3-pip``
+
+To get newer versions of pip (and also setuptools and wheel), you can enable the
+"unofficial" `PyPA Copr Repo <https://copr.fedoraproject.org/coprs/pypa/pypa/>`_
+using `these instructions
+<https://fedorahosted.org/copr/wiki/HowToEnableRepo>`__, and run the same
+commands as above.
+
+
+CentOS/RHEL
+~~~~~~~~~~~
+
+CentOS and RHEL don't offer pip in their core repositories.
+
+It's common practice to install pip from the `EPEL repository
```

Comment:
Correct, python34 in EPEL stil has [with_rewheel set to 0](http://pkgs.fedoraproject.org/cgit/python34.git/tree/python34.spec?h=epel7&id=2c1c983d5a4b8bc79e771cb8fd3f571b37d38d71#n17), which [disables the dependencies on pip and setuptools](http://pkgs.fedoraproject.org/cgit/python34.git/tree/python34.spec?h=epel7&id=2c1c983d5a4b8bc79e771cb8fd3f571b37d38d71#n783).  I'm guessing that once python34-pip and python34-setuptools are actually built, this macro can be set to 1.